pkg/pc/fields/fields.go

```diff
@@ -58,6 +59,29 @@ type PodCluster struct {
 	Annotations Annotations
 }
 
+func (pc *PodCluster) Equals(other *PodCluster) bool {
+	if pc == nil && other == nil {
```

Comment:
i'm fine with it, might just be nice for new fields being added to be automatically included in equality